### PR TITLE
add queryAll query operation

### PR DESCRIPTION
**Pull request by jster1357**

### CODE REVIEW

File: src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java [hunk 1]
```diff
@@ -76,6 +77,15 @@
...
 
+  @Name(SalesforceSourceConstants.PROPERTY_OPERATION)
+  @Description("If set to query, the query result will only return current rows. If set to queryAll, " +
+          "all records, including deletes will be sourced")
+  @Nullable
+  private String operation;
+
+  private static final String DEFAULT_OPERATION = "query";
+
+
```
Comment (rmstar) [inline src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java]: remove extra newline

**Review by rmstar — APPROVED** [review]
please squash commits after addressing nit

File: src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java (hunk 1 above)
Comment (jster1357) [reply src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java]: done

Comment (jster1357) [reply src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java]: done

**Pull request merged**